.github/workflows/rust.yml

```diff
@@ -370,7 +370,7 @@ jobs:
         'test')
           cargo llvm-cov nextest --lcov --output-path lcov-${{ matrix.target }}-${{ matrix.special_features }}.info \
               --partition count:${PARTITION}/4 \
-              --workspace --features "cairo-1-hints,  test_utils, ${{ matrix.special_features }}"
```

Comment:
Instead of adding the `cairo-0-secp-hints` feature here, could you add it to the special_features array at line 326 of this file?

Reply:
Resolved via https://github.com/lambdaclass/cairo-vm/pull/1829/commits/cb374e78da6536fbcdf8f6d931a2ee1f96996ea3